crates/uv-performance-flate2-backend/Cargo.toml

```diff
@@ -6,8 +6,8 @@ publish = false
 [lib]
 doctest = false
 
-[target.'cfg(not(any(target_arch = "s390x", target_arch = "powerpc64")))'.dependencies]
+[target.'cfg(not(any(target_arch = "s390x", target_arch = "powerpc64", target_os = "freebsd")))'.dependencies]
```

Comment:
This is a real behavioral change. I'm not sure if it's needed unless you're cross-building like I am? Seems fine though? Should I split into a separate PR for changelog?